docs/reference/query-rules/apis/put-query-rule.asciidoc

```diff
@@ -79,18 +82,18 @@ Required for all criteria types except `always`.
 (Required, object) The actions to take when the rule is matched.
 The format of this action depends on the rule type.
 
-Actions depend on the rule type.
-For `pinned` rules, actions follow the format specified by the <<query-dsl-pinned-query,Pinned Query>>.
-The following actions are allowed:
+Actions depend on the rule type..
+The following actions are allowed for `pinned` or `exclude` rules:
 
-- `ids` (Optional, array of strings) The unique <<mapping-id-field, document IDs>> of the documents to pin.
+- `ids` (Optional, array of strings) The unique <<mapping-id-field, document IDs>> of the specified documents.
 Only one of `ids` or `docs` may be specified, and at least one must be specified.
-- `docs` (Optional, array of objects) The documents to pin.
+- `docs` (Optional, array of objects) The specified documents.
```

Comment:
Is the max # of documents always hardcoded to 100? If so, I'd mention that here too